test_conformance/math_brute_force/binary_double.cpp

```diff
@@ -219,6 +219,7 @@ cl_int Test(cl_uint job_id, cl_uint thread_id, void *data)
     cl_double *r;
     cl_double *s;
     cl_double *s2;
+    cl_int copysign_test = 0;
```

Comment:
This could be a `bool` of course, but I appreciate that you're following what's in `binary_float.cpp`.  So let's leave it as a `cl_int`, no changes needed here!

(hopefully one day we can deduplicate the binary_*.cpp files, so having fewer differences between them helps)